.travis.yml

```diff
@@ -192,7 +192,7 @@ jobs:
         - .travis/install.sh
       before_script:
         - python setup.py build
-        - python setup.py compile_webui
+        - RAIDEN_NPM_MISSING_FATAL=1 python setup.py compile_webui
```

Comment:
This variable would be better in `env`

Reply:
I didn’t set it globally since it's only relevant for this one case